.travis.yml

```diff
@@ -26,6 +26,7 @@ before_install:
 install:
   - export GIT_FULL_HASH=`git rev-parse HEAD`
   - pip install -r requirements.txt
+  - pip install --upgrade cython
```

Comment:
why upgrade cython?

Reply:
to try and make it work with the nightly environment.  It is currently failing on a pandas bug (which is fixed on master and will be released soon)

Reply:
sounds good thanks!